Fix:

implicit declaration of function 'getpass' is invalid in C99

implicitly declaring library function 'snprintf' with type 'int (char *, unsigned long, const char *, ...)'

implicitly declaring library function 'strdup' with type 'char *(const char *)'
--- mkpasswd.c.orig	2023-07-24 15:41:53.000000000 -0500
+++ mkpasswd.c	2023-07-24 15:41:53.000000000 -0500
@@ -19,7 +19,6 @@
  */
 
 /* for crypt, snprintf and strcasecmp */
-#define _XOPEN_SOURCE 500
 #define _BSD_SOURCE 1
 #define _DEFAULT_SOURCE 1
 #define __EXTENSIONS__ 1
--- utils.c.orig	2025-10-12 20:37:20.000000000 -0500
+++ utils.c	2025-10-14 16:42:14.000000000 -0500
@@ -23,7 +23,7 @@
  */
 
 /* for strdup */
-#define _XOPEN_SOURCE 500
+#define _XOPEN_SOURCE 600
 
 /* System library */
 #include <stdio.h>
